### ISSUE TITLE
Diversity

### ISSUE THREAD

**Issue by alvations** [nlposs/NLP-OSS#15]
Following up on the discussion from https://twitter.com/alvations/status/911250035668484096, lets keep this issue up so that we don't lose track of it

**Comment by alvations** [comment]
Regarding diversity, we have https://github.com/nlposs/NLP-OSS/blob/master/Proposal-2020.md#demographic-diversity in the 2020 proposal. 

Also, we have drafted a [Code of Conduct](https://github.com/nlposs/NLP-OSS/blob/master/Code-of-Conduct.md) on top of the ACL anti-harassment policy

**Comment by alvations** [comment]
We're glad to announce that we've added 28 new PC members, of which at least 6 came from [WiNLP BIG community[(http://www.winlp.org/big-directory/). 

Thank you all PC members, we'll update you through emails to confirm your softconf usernames and more details on dates, reviews, etc. very soon!

---- 

Updates on the `Code of Conduct` (CoC), abiding by the ACL Professional Conduct Committee's advice, we'll not have a custom CoC for NLP-OSS. However, we've suggested to them to have more details as CoC from other conferences/open communities to 

 - add more explicitly listed examples of both acceptable/appropriate and unappropriate/unacceptable behaviors and 
 - "stop immediately" clause which some event/conferences adopts. "Participants asked to stop any inappropriate behavior are expected to comply immediately." E.g. https://us.pycon.org/2019/about/code-of-conduct/ , https://www.google.com/events/policy/anti-harassmentpolicy.html

The CoC drafted for the proposal can still be found on https://github.com/nlposs/NLP-OSS/blob/eac2cf9e45b51d73e87bad6ca02098f94647a87a/Code-of-Conduct.md
